add cypher parameter (#989)
vesoft-inc/nebula-docs: docs-2.0/nebula-console.md

  nebula> :param p2 => {"a":3,"b":false,"c":"Tim Duncan"};
  nebula> RETURN $p2.b AS b;
  +-------+
  | b     |
  +-------+
  | false |
  +-------+
  ```

- The command to view the saved parameters is as follows:

  ```ngql
  nebula> :params;
  ```

- The command to delete a specified parameter is as follows:

  ```ngql
  nebula> :param <param_name> =>;
  ```

## Export query results

Export query results,  which can be saved as a CSV file or DOT file.

!!! note

    - A file will be saved in the working directory, i.e., what the linux command `pwd` shows.

    - This command only works for the next query statement.

    - You can copy the contents of the DOT file and paste them in [GraphvizOnline](https://dreampuf.github.io/GraphvizOnline/) to generate a visualized execution plan.

- The command to export a csv file is as follows:

  ```ngql
  nebula> :CSV <file_name.csv>;
  ```

- The command to export a DOT file is as follows:

  ```ngql
  nebula> :dot <file_name.dot>
  ```

  The example is as follows:

  ```ngql
  nebula> :dot a.dot
  nebula> PROFILE FORMAT="dot" GO FROM "player100" OVER follow;
  ```

## Import a testing dataset

> The testing dataset is named `nba`. To view details about the schema and data, use the corresponding `SHOW` command.

Reviewer: I just modified the dataset name to basketballplayer.

```suggestion
The testing dataset is named `basketballplayer`. To view details about the schema and data, use the corresponding `SHOW` command.
```

The command to import a testing dataset is as follows:

```ngql

> nebula> :play nba

Reviewer: ```suggestion
nebula> :play basketballplayer
```

# Nebula Console

> Nebula Console is a native CLI client for Nebula Graph, it can be used to connect a Nebula Graph cluster and execute queries, while supporting special commands to manage parameters, export CSV files, export DOT files, import test datasets, etc. This topic describes these special commands.

Reviewer 2: ```suggestion
Nebula Console is a native CLI client for Nebula Graph. It can be used to connect a Nebula Graph cluster and execute queries. It can also support special commands to manage parameters, export CSV files, export DOT files, import test datasets, etc. This topic describes these special commands.
```
when "while" is used in a sentence as a conjunction, it has two meanings. mostly we regard it as a contrast. and better use short sentences here.

To connect Nebula Graph using Nebula Console, see [Step 3: Connect to Nebula Graph](2.quick-start/3.connect-to-nebula-graph.md).

!!! note

    The commands are case insensitive.

## Manage parameters

You can save parameters for parameterized queries.

!!! note

>     - Parameters are not supported in VID.

Reviewer 2: `参数化查询` and `参数` should be different things right? the CN doc uses the former, while here you use the latter

>     - Parameters are not supported in sample clause.

Reviewer 2: ditto

| v                                                  | n                                                     |
+----------------------------------------------------+-------------------------------------------------------+
| ("player100" :player{age: 42, name: "Tim Duncan"}) | ("player125" :player{age: 41, name: "Manu Ginobili"}) |
| ("player100" :player{age: 42, name: "Tim Duncan"}) | ("player101" :player{age: 36, name: "Tony Parker"})   |
+----------------------------------------------------+-------------------------------------------------------+
nebula> :param p2 => {"a":3,"b":false,"c":"Tim Duncan"};
nebula> RETURN $p2.b AS b;
+-------+
| b     |
+-------+
| false |
+-------+
```

The command to view the saved parameters is as follows:

```ngql
nebula> :params;
```

## Export a CSV file

CSV files save the returned result of a executed command.

!!! note

    - A CSV file will be saved in the working directory, i.e., what the linux command `pwd` shows.

    - This command only works for the next query statement.

The command to export a csv file is as follows:

```ngql

> nebula> :CSV <file_name.csv>

Reviewer 3: ```suggestion
nebula> :CSV <file_name.csv>;
```

Reviewer 3: add a semicolon at the end of a statement. please take a look at the full content

  ```

  The example is as follows:

  ```ngql
  nebula> :param p1 => "Tim Duncan";
  nebula> MATCH (v:player{name:$p1})-[:follow]->(n)  RETURN v,n;
  +----------------------------------------------------+-------------------------------------------------------+
  | v                                                  | n                                                     |
  +----------------------------------------------------+-------------------------------------------------------+
  | ("player100" :player{age: 42, name: "Tim Duncan"}) | ("player125" :player{age: 41, name: "Manu Ginobili"}) |
  | ("player100" :player{age: 42, name: "Tim Duncan"}) | ("player101" :player{age: 36, name: "Tony Parker"})   |
  +----------------------------------------------------+-------------------------------------------------------+
  nebula> :param p2 => {"a":3,"b":false,"c":"Tim Duncan"};
  nebula> RETURN $p2.b AS b;
  +-------+
  | b     |
  +-------+
  | false |
  +-------+
  ```

- The command to view the saved parameters is as follows:

  ```ngql
  nebula> :params;
  ```

- The command to delete a specified parameter is as follows:

  ```ngql
  nebula> :param <param_name> =>;
  ```

> ## Export a CSV file

Reviewer: Modify the two export-related subsections according to the CN source?
